server/chat-plugins/mafia.ts

```diff
@@ -1819,10 +1825,47 @@ export const pages: PageTable = {
 			buf += `<span id="mafia-lynches">`;
 			buf += game.lynchBoxFor(user.id);
 			buf += `</span>`;
-		} else if (game.phase === "night" && isPlayer) {
+		} else if (game.phase === "night" && isPlayer && !game.takeIdles) {
```

Comment:
(nit) this should probably be separated into `else if (game.phase === "night" && isPlayer) { if (!game.takeIdles) { ... } else { ... } }`